docs/runtime_suite/crud-service/30_encryption_configuration.md

```diff
@@ -135,7 +135,7 @@ try {
 
 #### How to generate a local master key using Shell Command
 ```shell
-cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 96 | head -n 1 > YOUR_KEY_PATH
+LC_CTYPE=C tr -dc 'a-zA-Z0-9' < /dev/urandom | fold -w 96 | head -1
```

Comment:
Why this locale set is necessary?

Reply:
I found out that without using it I get an error on OSx:

```
tr -dc 'a-zA-Z0-9' < /dev/urandom | fold -w 96 | head -1
tr: Illegal byte sequence
```

```
LC_CTYPE=C tr -dc 'a-zA-Z0-9' < /dev/urandom | fold -w 96 | head -1
GNtv2tuSW78LhUZ66U5UVHX7FbEq3eTZ6UVtRxGRgvHDrOTyCWpDKVYeXIMBJvZ6VoLQLjIJjmpFWHCtmcojl583squs3C3I
```

Seems like a problem with the locale default on MacOSx as explained here: https://stackoverflow.com/questions/19242275/re-error-illegal-byte-sequence-on-mac-os-x